content/content-strategy/identify-business-needs/what-business-needs/strategic-pyramid.md

```diff
@@ -0,0 +1,5 @@
+---
```

Comment:
!! File not listed in the `index.yml` so it won't appear on the page. Either remove this file or add its name to the index.yml at the appropriate spot.